pywhat/magic_numbers.py

```diff
@@ -12,7 +12,7 @@ def __init__(self):
 
     def open_file_loc(self, file_loc):
         with open(file_loc, "r", encoding="utf8", errors="ignore") as myfile:
-            r = myfile.readlines()
+            r = [myfile.read()]

```

Comment:
Doesn't this do the same thing?

Reply:
`readlines` returns a list with each line being a separate list item. While `read` returns the whole file as one item.